### PR TITLE
Feature mow session recycler view

### PR DESCRIPTION
**Pull request by LasseLejon**
Created a recyclerview for the mow session list item looking like this:
image

Feel free to think about what data that might be interesting to show here and how it can be presented better than what i've done currently.

### CODE REVIEW

File: app/src/main/java/se/ju/student/robomow/adapter/MowSessionAdapter.kt [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    private fun mowSessionIsComplete(mowSession: MowSession): Boolean {
+        return mowSession.end != null
+    }
+
+    private fun getMowingTimeMinutes(startTimeStamp: Long, endTimeStamp: Long): String {
+        val mowingTime = endTimeStamp - startTimeStamp
+        return mowingTime.div(60).toString()
+    }
+
+    private fun getDateFromTimeStamp(timeStamp: Long): String {
```
Comment (phille923) [inline app/src/main/java/se/ju/student/robomow/adapter/MowSessionAdapter.kt]: This getDateFromTimeStamp function does not convert to Swedish time-zone. The date is therefore incorrect by 2 hours.
A suggestion on how to solve it is using _ZoneId.systemDefault()_ to get the phones time zone.

File: app/src/main/java/se/ju/student/robomow/adapter/MowSessionAdapter.kt [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            return context.getString(R.string.mowing_status_complete)
+        }
+        return context.getString(R.string.mowing_status_active)
+    }
+
+    private fun mowSessionIsComplete(mowSession: MowSession): Boolean {
+        return mowSession.end != null
+    }
+
+    private fun getMowingTimeMinutes(startTimeStamp: Long, endTimeStamp: Long): String {
```
Comment (phille923) [inline app/src/main/java/se/ju/student/robomow/adapter/MowSessionAdapter.kt]: If you feel like doing it, I think displaying the time in hh:mm could be pretty cool. But that is just me being nitpicky

**Review by phille923 — APPROVED** [review]
Looks really good! 👑
I've added some feedback here in the review which you can look into. Otherwise everything else seems great!

File: app/src/main/java/se/ju/student/robomow/adapter/MowSessionAdapter.kt (hunk 1 above)
Comment (LasseLejon) [reply app/src/main/java/se/ju/student/robomow/adapter/MowSessionAdapter.kt]: Oh yeah I was looking into this but completely forgot adding it, good find!

File: app/src/main/java/se/ju/student/robomow/adapter/MowSessionAdapter.kt (hunk 2 above)
Comment (LasseLejon) [reply app/src/main/java/se/ju/student/robomow/adapter/MowSessionAdapter.kt]: Sure, this can be a better way of showing it.

**Pull request merged**